terraform/modules/jenkins/ssm.tf

```diff
@@ -1,25 +1,3 @@
-resource "aws_ssm_parameter" "access_key" {
```

Comment:
I've done `terraform state rm` to get terraform to forget about these resources.